registration/registration-services/src/main/java/io/mosip/registration/dao/impl/AuditDAOImpl.java

```diff
@@ -53,7 +53,7 @@ public List<Audit> getAudits(String registrationId, String timestamp) {
 
 		try {
 			List<Audit> audits;
-			if (timestamp == null) {
+			if (timestamp == null || timestamp.isEmpty()) {
```

Comment:
timestamp.isEmpty()
checks for both null and emptiness

Reply:
Corrected.